### ISSUE TITLE
Enable reporting to server list not working with Teamspeak Docker?

### ISSUE THREAD

**Issue by joe-joachim** [TeamSpeak-Systems/teamspeak-linux-docker-images#18]
I cannot get the Teamspeak to be listed on the directory. Can somebody help me?
I've reviewed the official support information on this issue.

I can access the Teamspeak server externally and the required ports are open.

The log says:
 |Query         |   |listening for query on 0.0.0.0:10011, [::]:10011
 |VirtualServer |1  |listening on 0.0.0.0:9987, [::]:9987
 |FileManager   |   |listening on 0.0.0.0:30033, [::]:30033

but I can still access it externally. Could it have anything to do with the listening IP not being correct?

Thanks for any and all replies.

**Comment by chrisTeamSpeak** [comment]
The IP should not be a problem.

Check following:
- Weblist reporting is enabled in the server
- Ports 2011-2110 UDP (out) are not blocked
- Wait at least 10 minutes before you check the weblist
- Search for the server name directly

https://support.teamspeakusa.com/index.php?/Knowledgebase/Article/View/324/0/how-do-i-add-or-remove-my-server-from-the-server-list

**Comment by joe-joachim** [comment]
Dear Chris,
Thank you for your reply.

The link you have provided me is exactly the the information I have read and gone through. The ports 2011-2110 is not being blocked. I have activated "Enable reporting to the server list" and it was confirmed the sever was edited. I have waited more than 10 minutes. I have also tried to search for the server by the full name either with "contains" or "starting with". It won't show up :(

Any other ideas?

Kind regards,
Joe
